### PR TITLE
BIP340: fix broken link to Schnorr's blind signature attack

### PR DESCRIPTION
**Pull request by meshcollider**
The link in BIP 340 to Schnorr's 2001 paper "Security of Blind Discrete Log Signatures Against Interactive Attacks" is broken.

This fixes it with an updated link (at the same domain).

### CODE REVIEW

File: bip-0340.mediawiki [hunk 1]
```diff
@@ -233,7 +233,7 @@ Adaptor signatures, beyond the efficiency and privacy benefits of encoding scrip
...
+A blind signature protocol is an interactive protocol that enables a signer to sign a message at the behest of another party without learning any information about the signed message or the signature. Schnorr signatures admit a very [http://publikationen.ub.uni-frankfurt.de/files/4292/schnorr.blind_sigs_attack.2001.pdf simple blind signature scheme] which is however insecure because it's vulnerable to [https://www.iacr.org/archive/crypto2002/24420288/24420288.pdf Wagner's attack]. A known mitigation is to let the signer abort a signing session with a certain probability, and the resulting scheme can be [https://eprint.iacr.org/2019/877 proven secure under non-standard cryptographic assumptions].
```
Comment (jonasnick) [inline bip-0340.mediawiki:236]: https?

Comment (meshcollider) [reply bip-0340.mediawiki:236]: That gives "This connection is not private" warnings due to obsolete TLS version unfortunately

Comment (MarcoFalke) [reply bip-0340.mediawiki:236]: I sent them an email

Comment (sipa) [reply bip-0340.mediawiki:236]: It's obviously unreasonable to expect an institution that does research around cryptography to actually *use* a cryptographic protocol.

**Comment by meshcollider** [comment]
So, considering a plaintext link is better than a broken one, let's get this merged :)

**Comment by luke-jr** [comment]
Need an ACK from @sipa @jonasnick or @real-or-random

**Review by real-or-random — APPROVED** [review]
ACK https://github.com/bitcoin/bips/pull/1275/commits/d58f2b29f7afcb1f72837c5283c329ee330c5889

**Pull request merged**